src/Directory.Build.props

```diff
@@ -12,21 +12,22 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Description>Akka Persistence journal and snapshot store backed by MongoDB database.</Description>
     <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <LangVersion>latest</LangVersion>
   </PropertyGroup>
   <PropertyGroup>
     <NetCoreTestVersion>netcoreapp3.1</NetCoreTestVersion>
-    <NetTestVersion>net6.0</NetTestVersion>
+    <NetTestVersion>net7.0</NetTestVersion>
     <NetFrameworkTestVersion>net471</NetFrameworkTestVersion>
     <NetStandardLibVersion>netstandard2.0</NetStandardLibVersion>
 
+    <AkkaVersion>1.5.8</AkkaVersion>
     <XunitVersion>2.4.2</XunitVersion>
     <TestSdkVersion>17.6.3</TestSdkVersion>
-    <AkkaVersion>1.5.8</AkkaVersion>
-	<FluentAssertionsVersion>6.11.0</FluentAssertionsVersion>
+    <FluentAssertionsVersion>6.11.0</FluentAssertionsVersion>
   </PropertyGroup>
   <!-- SourceLink support for all Akka.NET projects -->
   <ItemGroup>
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.SourceLink.GitHub" PrivateAssets="All" />
```

Comment:
Note I have seen some other repos that have moved to central package management have the `Microsoft.SourceLink.GitHub` in the `Directory.Packages.props` file but no reference either in the csproj or `Directory.Build.props`

I have noticed it in `Akka.Persistence.Sql` and `Akka.Persistence.PostgreSql` repos